cognee/tests/unit/processing/chunks/chunk_by_word_test.py

```
@@ -0,0 +1,40 @@
+        INPUT_TEXTS["english_lists"],
+        INPUT_TEXTS["python_code"],
+        INPUT_TEXTS["chinese_text"],
+    ],
+)
+def test_chunk_by_word_splits(input_text):
+    chunks = np.array(list(chunk_by_word(input_text)))
+    space_test = np.array([" " not in chunk[0].strip() for chunk in chunks])
+
+    assert np.all(
+        space_test
+    ), f"These chunks contain spaces within them: {chunks[space_test == False]}"
```

Ruff

Avoid equality comparisons to `False`; use `if not space_test:` for false checks

Replace with `not space_test`

(E712)